src/sec_newton.mbx

```diff
@@ -558,222 +557,577 @@
+            </var>
+            <var name="$x[2]">
+              <static>0.9133412072</static>
+            </var>
+            <var name="$x[3]">
+              <static>0.9961317034</static>
+            </var>
+            <var name="$x[4]">
+              <static>0.9999925085</static>
+            </var>
+            <var name="$x[5]">
+              <static>1</static>
             </var>
             <pg-code>
+              $f=list_random('ln(x)');
+              $f=Compute($f);
+              Context("LimitedNumeric");
+              $x[0]=random(2,2,0.1);
+              $df=$f->D('x');
+              for my $i (1..5) {
+              $x[$i] = Compute($x[$i-1] - ($f->eval(x=>$x[$i-1]))/($df->eval(x=>$x[$i-1])));
+              };
             </pg-code>
           </setup>
           <statement>
-            <p><m>f(x) = \ln(x)</m>, <m>x_0=2</m></p>
+            <p>Let <m>f(x) = <var name="$f" /></m>. Use five iterations of Newton's Method with <m>x_0=<var name="$x[0]" /></m> to approximate the root.</p>
+            <table>
+              <tabular>
+                <row>
+                  <cell><m>x_0=</m></cell>
+                  <cell><m><var name="$x[0]" /></m></cell>
+                </row>
+                <row>
+                  <cell><m>x_1=</m></cell>
+                  <cell><var name="$x[1]" width="10"/></cell>
+                </row>
+                <row>
+                  <cell><m>x_2=</m></cell>
+                  <cell><var name="$x[2]" width="10"/></cell>
+                </row>
+                <row>
+                  <cell><m>x_3=</m></cell>
+                  <cell><var name="$x[3]" width="10"/></cell>
+                </row>
+                <row>
+                  <cell><m>x_4=</m></cell>
+                  <cell><var name="$x[4]" width="10"/></cell>
+                </row>
+                <row>
+                  <cell><m>x_5=</m></cell>
+                  <cell><var name="$x[5]" width="10"/></cell>
+                </row>
+              </tabular>
+            </table>
+            <p>Compare <m>x_5</m> to the known value of the root.</p>
           </statement>
-          <solution>
-            <p><m>x_0=2</m>, <m>x_1=0.6137056389</m>, <m>x_2=0.9133412072</m>, <m>x_3=0.9961317034</m>, <m>x_4=0.9999925085</m>, <m>x_5=1</m></p>
-          </solution>
         </webwork>
       </exercise>
     </exercisegroup>
 
+<!--IN THIS GROUP, FIND ROOTS OF FUNCTIONS, POSSIBLY ON AN INTERVAL-->
     <exercisegroup>
       <introduction>
         <p>In the following exercises, use Newton's Method to approximate all roots of the given functions accurate to three places after the decimal. If an interval is given, find only the roots that lie in that interval. Use technology to obtain good initial approximations.</p>
       </introduction>
+      
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$f">
```

Comment:
Yes. It's best not to think of the static entries as something special because they are "static". Instead, think of them as math mode content. So since `x^2\cos(x) + (x-1)\sin(x)` is the right way to do this in general math mode, it is the right way to do this here.